filer/fields/folder.py

```diff
@@ -122,6 +123,8 @@ def __init__(self, **kwargs):
                 )
                 warnings.warn(msg, SyntaxWarning)
         kwargs['to'] = dfl
+        if DJANGO_VERSION >= (1, 9):
```

Comment:
current versions of django-filer do not support Django<1.11, so make a case distinction is unnecessary here.